otter/convergence/service.py

```diff
@@ -5,6 +5,85 @@
 :obj:`Converger`.
 """
 
+# # Note [Convergence cycles]
+#
+# A very abstract version of our convergence cycle:
+# - CYCLE (every N seconds)
+# - find all of my divergent groups
```

Comment:
Non-blocking:  might want to note that this is done by looking at divergent flags, which is implied but not overtly stated.